HighEnergyObsCoreExt.tex

```diff
@@ -415,48 +421,49 @@ \subsubsection{Clarification of ``flux'' in some term definitions}
 
 These are not all covered by the UCD vocabulary. This would be particularly useful for cases where we actually should have the same UCDs as {\em e.g.\/} a Radio measurement, but where we use very different units. For example: Jansky and erg/cm2/s/TeV are both types of "spectral flux density", but the unit analysis doesn't agree due to frequency-energy equivalency.
 
-Restating the descriptions to explicitly state ``Particle or energy flux or magnitude'' where appropriate would resolve this ambiguity associated to unit confusions. These different type of fluxes will be proposed to be added in the UCD list.
+Restating the descriptions based on dimensions and not units, and this for photons and particles, would resolve this ambiguity associated to unit confusions. As consequence, a refinement of the UCD list is proposed in the section \ref{sec:UCDs}.
```

Comment:
Perhaps change "Restating the descriptions based on dimensions and not units, and this for photons and particles" to "Restating the descriptions for photons and particles based on dimensions and not units".